packages/react-components/src/utils/number.ts

```diff
@@ -0,0 +1,30 @@
+import { ThresholdDataTypes } from '../common/dataTypes';
+
+const MAX_PRECISION = 4;
+
+/**
+ * Rounds a number to a pre-determined precision
+ *
+ * i.e. round(100000.1234) => 100000.1234
+ *      round(100000.12345678) => 100000.1234
+ *      round(.02345678) => 0.02346
+ */
+export const round = (num: number): number => {
```

Comment:
Do we not want to use lodash round here?

Reply:
willing to re-evaluate that when we tackle customization of decimal precision